pkgs/servers/sql/pgpool/default.nix

```diff
@@ -34,6 +34,12 @@ stdenv.mkDerivation rec {
     "sysconfdir=\${out}/etc"
   ];
 
+  patches = lib.optionals (stdenv.isDarwin) [
```

Comment:
Do we need to guard it against Darwin, does it work for x86?

Reply:
It should work on Intel as well.
The patch is a noop for non-darwin, so technically we don't need the optional. I added it more as a comment for where the psych applies.